cmd/helm/template.go

```diff
@@ -103,6 +104,7 @@ func newTemplateCmd(out io.Writer) *cobra.Command {
 	f.StringArrayVar(&t.fileValues, "set-file", []string{}, "set values from respective files specified via the command line (can specify multiple or separate values with commas: key1=path1,key2=path2)")
 	f.StringVar(&t.nameTemplate, "name-template", "", "specify template used to name the release")
 	f.StringVar(&t.kubeVersion, "kube-version", defaultKubeVersion, "kubernetes version used as Capabilities.KubeVersion.Major/Minor")
+	f.StringArrayVarP(&t.apiVersions, "api-version", "a", []string{}, "kubernetes api versions used for Capabilities.APIVersions")
```

Comment:
This should probably be plural (`api-versions`)

Reply:
@thomastaylor312 done!